tests/test_project.py

```diff
@@ -1836,6 +1868,171 @@ class TestGroupDynamicProjectMainInterface(TestProjectMainInterface):
     project_class = _DynamicTestProject
 
 
+class TestAggregatesProjectBase(TestProjectBase):
+    project_class = _AggregateTestProject
+    entrypoint = dict(
+        path=os.path.realpath(
+            os.path.join(os.path.dirname(__file__), "define_aggregate_test_project.py")
+        )
+    )
+
+    def mock_project(self):
+        project = self.project_class.get_project(root=self._tmp_dir.name)
+        for i in range(1, 31):
+            project.open_job(dict(i=i, even=bool(i % 2 == 0))).init()
+        project = project.get_project(root=self._tmp_dir.name)
+        project._entrypoint = self.entrypoint
+        return project
+
+    @pytest.fixture(autouse=True)
+    def setup_main_interface(self, request):
+        self.project = self.mock_project()
+        os.chdir(self._tmp_dir.name)
+        request.addfinalizer(self.switch_to_cwd)
+
+
+class TestAggregatesProjectUtilities(TestAggregatesProjectBase):
+    def test_AggregatesCursor(self):
+        project = self.mock_project()
+        agg_cursor = _AggregatesCursor(project=project)
+        assert agg_cursor._project == project
```

Comment:
```suggestion
        assert agg_cursor._project is project
```

This is really an identity check isn't it?

Reply:
I agree. Committing.